src/webview/openshift-terminal/app/terminalInstance.tsx

```diff
@@ -35,9 +132,32 @@ export const TerminalInstance = (props: {
         newTerm.loadAddon(new WebLinksAddon());
         newTerm.loadAddon(new WebglAddon());
         newTerm.loadAddon(fitAddon);
+        newTerm.attachCustomKeyEventHandler((keyboardEvent: KeyboardEvent) => {
+            // Copy/Paste/Select All keybinding handlers
+            if (keyboardEvent.shiftKey && keyboardEvent.ctrlKey) {
+                // https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values
+                if (keyboardEvent.code === 'KeyC' && term.hasSelection) {
+                    // Ctrl+Shift+C copies
+                    void navigator.clipboard.writeText(term.getSelection());
```

Comment:
The problem here is that the KeyboardEvent processing is not stopped after the `Ctrl-Shift-C`/`Ctrl-Shift-A` hot keys are processed here. For example, on my Fedora,  pressing `Ctrl-Shift-C`  leads into copying the selected text into Clipboard (this handler) plus opening a System Terminal.
 
Everything else works fine,

Reply:
I'm not too sure what I can do to prevent this. I tried adding `preventDefault()` in the key event handler, but that didn't fix it.

Reply:
Alright I pushed the fix, let me know if it works for you